I could not use this with any version other than 3.7 (tried 2.7, 3.5,… (#2384)
iterative/dvc.org: content/docs/use-cases/versioning-data-and-model-files/tutorial.md

## Preparation

> > We have tested our tutorials and examples with Python 3.7. We don't recommend
> > using earlier or later versions.

Reviewer: Maybe just
```suggestion
> We have last tested this tutorial with Python 3.7.
```
But I need to confirm this info. It's quite possible, as this one tutorial hasn't been tried in a while...
